package.json

```diff
@@ -1,12 +1,18 @@
 {
   "name": "frontend-coding-standards",
   "version": "1.0.0",
+  "private": false,
   "description": "MediaMonks - Frontend Coding Standards",
   "repository": "git@github.com:mediamonks/frontend-coding-standards.git",
-  "author": "Thijs <thijs@mediamonks.com>",
   "license": "MIT",
-  "private": false,
+  "author": "Thijs <thijs@mediamonks.com>",
   "scripts": {
-    "generate-table-of-contents": "node ./tools/generate-table-of-contents.js"
+    "generate-table-of-contents": "node ./tools/generate-table-of-contents.js",
+    "format": "prettier --write '**/*.js'"
```

Comment:
Wouldn't it be a good idea to format all file formats (JSON, Markdown etc.) that are supported by prettier?